src/GovUk.Education.ExploreEducationStatistics.Content.Api/Controllers/DataSetFilesController.cs

```diff
@@ -39,6 +40,7 @@ public async Task<ActionResult<PaginatedListViewModel<DataSetFileSummaryViewMode
                 themeId: request.ThemeId,
                 publicationId: request.PublicationId,
                 releaseVersionId: request.ReleaseId,
+                geographicLevel: request.GeographicLevelEnum,
```

Comment:
Just wondering if from an API point of view you considered implementing this to allow an array of levels rather than only allowing one value? I think it would have provided more flexibility and been a similar amount of work (although I might be wrong about that!)